docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+networks:
+  overlay:
+
+services:
+  mysql:
+    image: mysql:8-debian
```

Comment:
`8-debian` was last [pushed a year ago](https://hub.docker.com/_/mysql/tags?name=8-debian). This should be `8.0` due to the `mysql_native_password` issue, as `8` would be equivalent to `8.4`.

I've experienced issue in the past when upgrading, so it is helpful to use `8.0.41` and manually update when new releases come out. Even if the version lags behind, product is not using this.

Alternatively it could be changes to `bitnami/mysql:8.0.40`.

Reply:
Thank you. I see `bitnami/mysql:8.3` is in use in the latest version of the file. This is an [Innovation Release](https://dev.mysql.com/doc/refman/8.4/en/mysql-releases.html) so is no longer supported rather than either 8.0 or 8.4 which are both still supported. [As far as I know](https://dba.stackexchange.com/a/339256/184665) `--default-authentication-plugin=mysql_native_password` won't work with 8.4. Instead use `--mysql-native-password=ON` and maybe `--default-auth=mysql_native_password` as [documented here](https://dev.mysql.com/doc/refman/8.4/en/native-pluggable-authentication.html).